src/Components/CreateImageWizard/CreateImageWizard.tsx

```diff
@@ -437,6 +437,13 @@ const CreateImageWizard = ({ isEdit }: CreateImageWizardProps) => {
                 name='Azure'
                 id='wizard-target-azure'
                 key='wizard-target-azure'
+                status={
+                  !isAzureTenantGUIDValid(azureTenantId) ||
+                  !isAzureSubscriptionIdValid(azureSubscriptionId) ||
+                  !isAzureResourceGroupValid(azureResourceGroup)
+                    ? 'error'
+                    : 'default'
+                }
```

Comment:
We did not have this, and I believe this might help navigate users that used to have sources for Azure target  realize that they need to change this. I also believe we might want to add this to other Targets, so that there is an error in case something is missing.